### PR TITLE
Court enhancements

### LINKED ISSUE #2135: [Court] Display APR
We should display APR as it's a standard metric in DeFi for users to get an idea of returns. Probably we can replace one of these to make space:
image
Calculation for APR would be (inflation amount)/(total staked)

### LINKED ISSUE #2136: [Court] Show juror details
We have a component to display more details on jurors which is useful for users to select the one to delegate to

### LINKED ISSUE #2137: [Court] Show juror identities where possible

### PR DESCRIPTION
**Pull request by Robiquet**
closes #2135 
closes #2136
closes #2137

### CODE REVIEW

File: components/court/JurorsTable.tsx [hunk 1]
```diff
@@ -32,44 +34,55 @@ const columns: TableColumn[] = [
     accessor: "status",
     type: "text",
   },
-  {
-    header: "",
-    accessor: "button",
-    type: "component",
-    width: "150px",
-  },
```
Comment (robhyrk) [inline components/court/JurorsTable.tsx:40]: can this be removed?

File: lib/hooks/queries/useIdentities.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  const [sdk, id] = useSdkv2();
+
+  const query = useQuery(
+    [id, identitiesRootKey, addresses],
+    async () => {
+      if (addresses && isRpcSdk(sdk)) {
+        const userIdentities: (UserIdentity | null)[] = [];
+        const res = await sdk.api.queryMulti(
+          addresses.map((address) => [
+            sdk.api.query.identity.identityOf as any,
```
Comment (robhyrk) [inline lib/hooks/queries/useIdentities.ts:18]: is it possible to use a more defined type instead of `any` ?

**Review by robhyrk — APPROVED** [review]
looks good! just left a few suggestions

File: components/court/JurorsTable.tsx (hunk 1 above)
Comment (Robiquet) [reply components/court/JurorsTable.tsx:40]: Wanted to check with Jorn, we might keep it

File: lib/hooks/queries/useIdentities.ts (hunk 2 above)
Comment (Robiquet) [reply lib/hooks/queries/useIdentities.ts:18]: The api doesn't seem to have a type for this for some reason. I could write one, though the substrate types are a bit of a pain to write out manually and it wouldn't bring us much additional safety.

**Review by yornaath — APPROVED** [review]

**Pull request merged**